schema/deploy/tables/certification_url.sql

```diff
@@ -18,7 +18,8 @@ create table ggircs_portal.certification_url (
   updated_by int references ggircs_portal.ciip_user,
   deleted_at timestamp with time zone,
   deleted_by int references ggircs_portal.ciip_user,
-  certification_request_sent_to varchar(1000),
+  send_certification_request boolean not null default false,
```

Comment:
UX, maybe already addressed: if this is false by default how do we ensure users know that this critical email won't be sent unless the toggle is clicked?

Out of scope on this PR but worth closing the loop with @rajpersram maybe?

Reply:
Maybe the default should be true and the toggle should begin as clicked? I'll make a task in youtrack to visit the idea with @rajpersram

Reply:
https://youtrack.button.is/issue/GGIRCS-1492